agent/cmd/determined-agent/init.go

```diff
@@ -152,6 +152,6 @@ func registerAgentConfig() {
 	registerInt(flags, name("agent-reconnect-backoff"), defaults.AgentReconnectBackoff,
```

Comment:
I think we are going to need a lot of changes to the circleci file to no longer run agent tests